repo/fsrepo_test.go

```diff
@@ -342,6 +342,77 @@ func TestRepoAPIFile(t *testing.T) {
 	})
 }
 
+func TestCreateRepo(t *testing.T) {
+	cfg := config.NewDefaultConfig()
+
+	t.Run("folder exists", func(t *testing.T) {
```

Comment:
i expected this to be an error case, based on the name. maybe: successfully creates if dir exists